packages/orchestrator-ui-components/src/pages/metadata/WfoWorkflowsPage.tsx

```diff
@@ -52,6 +57,10 @@ export type WorkflowListItem = Pick<
     productTags: string[];
 };
 
+type WorkflowListExportItem = Omit<WorkflowDefinition, 'products'> & {
```

Comment:
Since export is/should be based on the type above the `WorkflowListItem`. Maybe make it pick and omit from there. The exported data should be very similar to the data we see in the table.

Reply:
done